src/pages/login/SignIn.vue

```diff
@@ -0,0 +1,131 @@
+<template lang='pug'>
```

Comment:
where is used this page???

Reply:
Is not used on the UI. I'll remove the component.